### PR TITLE
Github actions CI

### LINKED ISSUE #89: Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by keynmol**
Closes #89 

:man_shrugging: Something seems to be running. It shouldn't be this simple :D 

https://github.com/keynmol/sbt-explicit-dependencies/runs/1467709269

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          - sbt_version: 1.2.8
+            test_groups: 'basic/*'
+          - sbt_version: 1.3.13
+            test_groups: 'basic/* 1.3+/*'
+          - sbt_version: 1.4.0
+            test_groups: 'basic/* 1.3+/*'
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (cb372) [inline .github/workflows/ci.yml:21]: Let's bump this to v2

**Comment by cb372** [comment]
Nice! Not sure why the workflow didn't run against the PR though. Maybe it doesn't start working until the workflows file is in the master branch?

You may as well delete .travis.yml as part of this PR.

**Comment by keynmol** [comment]
@cb372 not sure tbh, may be there are some settings that need to be changed for PRs from forks?

**Comment by keynmol** [comment]
I'm also going to add coursier cache action, as I don't think the dependencies of this plugin are likely to change very often and it should help save time on builds

**Review by cb372 — APPROVED** [review]
Let's merge and see what happens

**Comment by keynmol** [comment]
Seems like it works 🎉 https://github.com/cb372/sbt-explicit-dependencies/actions/runs/393834960

**Pull request merged**